userspace/libsinsp/capture_stats_source.h

```diff
@@ -43,4 +44,18 @@ class SINSP_PUBLIC capture_stats_source
 	 * @param[out] stats The capture statistics
 	 */
 	virtual void get_capture_stats(scap_stats* stats) const = 0;
+
+	/**
+	 * Get engine statistics (including counters and `bpftool prog show` like stats).
+	 *
+	 * @note Won't work on file captures.
```

Comment:
> Won't work on file captures.

That's an implementation detail, we never now what we can come up with in the future.